src/cli/runCli.ts

```diff
@@ -36,15 +42,13 @@ export const runCli = async (
         return ResultStatus.Succeeded;
     }
 
-    let result: TSLintToESLintResult;
+    let result: TSLintToESLintResult = {
+        errors: [new Error("No configurations provided")],
+        status: ResultStatus.Failed,
+    };
 
-    try {
-        result = await dependencies.convertConfig(parsedArgv);
-    } catch (error) {
-        result = {
-            errors: [error as Error],
-            status: ResultStatus.Failed,
-        };
+    for (const convertConfig of dependencies.convertConfigs) {
+        result = await tryConvertConfig(convertConfig, parsedArgv);
```

Comment:
Reassigning `result` means if the first one fails you'll end up not reporting that error. Unfortunate. How about pushing to `result.errors` if need be, and having `status` be whether there are multiple errors?

Reply:
Reporting the errors to console and immediately return if failed, otherwise succeed.